packages/yew/src/callback.rs

```diff
@@ -82,4 +82,70 @@ impl<IN: 'static, OUT: 'static> Callback<IN, OUT> {
     }
 }
 
+impl<IN: 'static> Callback<IN> {
+    /// Creates a new callback from another callback and a function
+    /// That when emitted will call that function and will emit the original callback if it is
+    /// not `None`.
```

Comment:
```suggestion
    /// Creates a new callback from another callback and a function.
    /// When emitted will call the function and, only if it returns `Some(value)`, will emit
    /// `value` to the original callback.
```

Double check the formatting, since the suggestion comes from the online form, but I think the rephrasing is more readable.